### PR TITLE
Adding wildcard suffix to SCVBYOATenantStack under CloudformationAccess Sid

### PR DESCRIPTION
**Pull request by arijit725**
Adding wildcard suffix to SCVBYOATenantStack under CloudformationAccess Sid

### CODE REVIEW

**Comment by jinalkathiara** [comment]
Can you update your comment and PR commit message to reflect that this change is required for the contact center 14.1 onwards ?

File: iam_policies/SCVProvisioningPolicy.json [hunk 1]
```diff
@@ -162,7 +162,7 @@
                 "arn:aws:kinesis:*:<AWS_ACCOUNT_ID>:stream/*-<AWS_ACCOUNT_ID>-ContactLensStream",
                 "arn:aws:cloudformation:*:<AWS_ACCOUNT_ID>:stack/*ContactCenterStack/*",
                 "arn:aws:cloudformation:*:<AWS_ACCOUNT_ID>:stack/*ContactCenterStack*/*",
-                "arn:aws:cloudformation:*:<AWS_ACCOUNT_ID>:stack/SCVBYOATenantStack/*",
+                "arn:aws:cloudformation:*:<AWS_ACCOUNT_ID>:stack/SCVBYOATenantStack*/*",
```
Comment (jinalkathiara) [inline iam_policies/SCVProvisioningPolicy.json:165]: Update PR Commit message

**Review by jinalkathiara — CHANGES_REQUESTED** [review]

File: iam_policies/SCVProvisioningPolicy.json (hunk 1 above)
Comment (arijit725) [reply iam_policies/SCVProvisioningPolicy.json:165]: There is already commit message present. Please let me know if anything specific need to be included.

Comment (jinalkathiara) [reply iam_policies/SCVProvisioningPolicy.json:165]: Can you add commit message for the open source friendly language. "Updating Contact Center with 14.1 provisioning role"

**Review by jinalkathiara — APPROVED** [review]

**Pull request merged**